docs/faas/faas-metrics.md

```diff
@@ -1,5 +1,5 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: FaaS
+linkTitle: Metrics
```

Comment:
Should this be FaaS Metrics ?

Reply:
The given title results in this side nav:

> <img width="283" alt="image" src="https://github.com/open-telemetry/semantic-conventions/assets/4140793/75aa59d3-c4fb-4b59-b1d6-84c95787cd4f">

It seems redundant to include "FaaS" in a title under the "FaaS" section, but it's your call.

Reply:
Ah, got it.  SO contextually, the title is always under the previous title.  LGTM